src/client/src/components/pages/participants/addParticipant/staticData.js

```diff
@@ -183,6 +192,7 @@ export const fields = [
       name: "ni_number",
```

Comment:
Does this need extra validation as per Kathryn's suggestions? NI Numbers are two letters, six digits, and one letter.

**Valid NI Number**: JZ123456B
**Invalid NI Number**: JZD1243456BB

Maybe this needs a regex validation. https://regexr.com/